### ISSUE TITLE
Chain Sync Info

### ISSUE THREAD

**Issue by dhyaniarun1993** [ledgerwatch/erigon#6747]
Hi,
I am new to Erigon and I have some questions. I am posting it here since the Discord invite link isn't working for me. 

I have a geth node running with dev chain. I would like to sync Erigon using this node. I tried using staticPeer but it doesn't show any sync progress. I hope someone can help me with this.

Thank you

**Comment by AskAlexSharov** [comment]
Erigon logs plz

**Comment by AskAlexSharov** [comment]
Discord: send email to tg [at] torquem.ch 
Or pm in twitter

**Comment by dhyaniarun1993** [comment]
> Erigon logs plz

Thanks Alex for reverting. Please find the details below

Startup command 
```
./build/bin/erigon --datadir=dev --private.api.addr=localhost:9090 --port=30304 --port=30302 --authrpc.port=8550  --http.port=8546  --nodiscover  --staticpeers=enode://2c8f6d4764c3aca75696e18aeef683932a2bfa0be1603adb54f30dfad8e5cf2372a9d6eeb0e5caffba1fca22e12878c450e6ef09434888f04c6a97b6f50c75d4@127.0.0.1:30301
```

```
WARN[01-31|10:01:46.023] no log dir set, console logging only 
INFO[01-31|10:01:46.023] Initializing firehose 
INFO[01-31|10:01:46.024] Firehose initialized                     enabled=true sync_instrumentation_enabled=true mining_enabled=false block_progress_enabled=false compaction_disabled=false archive_blocks_to_keep=0 genesis_provenance="Geth Specific Flag" firehose_version=2.1 erigon_version=2.35.0-stable chain_variant=erigon
FIRE INIT 2.1 erigon 2.35.0-stable-63f1b12e
WARN[01-31|10:01:46.030] no log dir set, console logging only 
INFO[01-31|10:01:46.030] Build info                               git_branch=fix/modules git_tag=v2.35.1-15-g63f1b12ef-dirty git_commit=63f1b12ef5b11c0109f09a35f2d33fc01d6f1c7d
INFO[01-31|10:01:46.031] Starting Erigon on Ethereum mainnet... 
INFO[01-31|10:01:46.033] Maximum peer count                       ETH=100 total=100
INFO[01-31|10:01:46.033] starting HTTP APIs                       APIs=eth,debug,net,trace,web3,erigon
INFO[01-31|10:01:46.033] torrent verbosity                        level=WRN
INFO[01-31|10:01:46.034] Set global gas cap                       cap=50000000
INFO[01-31|10:01:46.073] [Downloader] Runnning with               ipv6-enabled=true ipv4-enabled=true download.rate=16mb upload.rate=4mb
INFO[01-31|10:01:46.074] Opening Database                         label=chaindata path=/Users/arundhyani/Stuff/Project/chainsafe/go-erigon/dev/chaindata
INFO[01-31|10:01:46.120] Initialised chain configuration          config="{ChainID: 1515, Homestead: 0, DAO: <nil>, DAO Support: true, Tangerine Whistle: 0, Spurious Dragon: 0, Byzantium: 0, Constantinople: 0, Petersburg: 0, Istanbul: 0, Muir Glacier: 0, Berlin: <nil>, London: <nil>, Arrow Glacier: <nil>, Gray Glacier: <nil>, Terminal Total Difficulty: 10790000, Merge Netsplit: <nil>, Shanghai: <nil>, Cancun: <nil>, Engine: clique}" genesis=0xc5902001cfab2bfb6cbfb5e752ab11be44af47d50dbd777a02f7098c169e780b
INFO[01-31|10:01:46.121] Effective                                prune_flags= snapshot_flags= history.v3=false
INFO[01-31|10:01:46.123] Initialising Ethereum protocol           network=1
INFO[01-31|10:01:46.143] Starting private RPC server              on=localhost:9090
INFO[01-31|10:01:46.143] new subscription to logs established 
INFO[01-31|10:01:46.168] [Sentinel] Sentinel started              enr=enr:-Ly4QBgv8rFxGbsZmg4eHsF-vUTNOmRhEsqy2f1ufSHELdivV19EbUbYiV-oCyw9sk-VRQ5DTt0b7AvxGA8xHxq92TgBh2F0dG5ldHOIAAAAAAAAAACEZXRoMpBKJsWLAgAAAP__________gmlkgnY0gmlwhH8AAAGJc2VjcDI1NmsxoQK6jP9XOdgfAs5R_6EUCzEOkgNYsxI5xOVN1WvZM1z7XohzeW5jbmV0cwCDdGNwgg-hg3VkcIIPoA
INFO[01-31|10:01:46.171] [Checkpoint Sync] Requesting beacon state uri=https://mainnet-checkpoint-sync.stakely.io/eth/v2/debug/beacon/states/finalized


INFO[01-31|10:03:39.594] [Checkpoint Sync] Retrieving lightclient bootstrap from sentinel root=8c87e830ad746c0e2ed040fce164e7e6ad97ccd0f5f6b4f10297aa70e8631311
INFO[01-31|10:03:46.123] [p2p] GoodPeers                          eth66=0 eth67=0
INFO[01-31|10:03:47.982] Store Initialized successfully           slot=5692352 root=38a496e664f9be90fb3e717c041e320be85511c8187ac4146db6e7e1d9234363
INFO[01-31|10:03:47.984] [Lightclient] Fetching Sync Committee Period period=694
INFO[01-31|10:03:47.984] [LightClient Gossip] Started Gossip 
INFO[01-31|10:03:47.986] rpc filters: subscribing to Erigon events 
INFO[01-31|10:03:47.987] new subscription to newHeaders established 
INFO[01-31|10:03:47.987] New txs subscriber joined 
INFO[01-31|10:03:47.992] Reading JWT secret                       path=/Users/arundhyani/Stuff/Project/chainsafe/go-erigon/dev/jwt.hex
INFO[01-31|10:03:47.992] Generated JWT secret                     path=/Users/arundhyani/Stuff/Project/chainsafe/go-erigon/dev/jwt.hex
INFO[01-31|10:03:47.994] [txpool] Started 
INFO[01-31|10:03:47.995] Started P2P networking                   version=66 self="enode://2b3943169e8db1f16b91e18c39f81f05fd5e6e25753f60bf8197b9e7893c76fa983b6add483013a9978e97ec67ce67fee0cd77fd2aec7cb8d32f781a07683c2d@127.0.0.1:30302?discport=0" name=erigon/v2.35.0-stable-63f1b12e/darwin-amd64/go1.19.1
INFO[01-31|10:03:47.995] HTTP endpoint opened for Engine API      url=localhost:8550 ws=true ws.compression=true
INFO[01-31|10:03:47.996] Started P2P networking                   version=67 self="enode://2b3943169e8db1f16b91e18c39f81f05fd5e6e25753f60bf8197b9e7893c76fa983b6add483013a9978e97ec67ce67fee0cd77fd2aec7cb8d32f781a07683c2d@127.0.0.1:30303?discport=0" name=erigon/v2.35.0-stable-63f1b12e/darwin-amd64/go1.19.1
INFO[01-31|10:03:47.997] HTTP endpoint opened                     url=localhost:8546 ws=true ws.compression=true grpc=false
INFO[01-31|10:03:48.001] [2/16 Headers] Waiting for Consensus Layer... 
INFO[01-31|10:04:15.162] [2/16 Headers] Handling new payload      height=16524000 hash=0xedfd1445d2799ac0665a474e65381a8d641a7f3241420481e53f33be345d7b86
INFO[01-31|10:04:15.165] [2/16 Headers] Downloading PoS headers... height=16523999 hash=0x27a5d6240acc3dd7e57a64d8fbf4d3af01d51fec37869cd7a8939a65ea06ee83 requestId=1
INFO[01-31|10:04:47.984] [Downloader] Downloaded PoS Headers      now=16523999 blk/sec=0.000
INFO[01-31|10:04:47.984] [txpool] stat                            block=0 pending=0 baseFee=0 queued=0 alloc=85.6MB sys=380.9MB
INFO[01-31|10:04:47.986] [LightClient] P2P                        peers=2
INFO[01-31|10:05:17.985] [Downloader] Downloaded PoS Headers      now=16523999 blk/sec=0.000
INFO[01-31|10:05:46.123] [p2p] GoodPeers                          eth66=0 eth67=0
INFO[01-31|10:05:47.984] [txpool] stat                            block=0 pending=0 baseFee=0 queued=0 alloc=85.6MB sys=380.9MB
INFO[01-31|10:05:47.984] [Downloader] Downloaded PoS Headers      now=16523999 blk/sec=0.000
INFO[01-31|10:05:47.985] [LightClient] P2P                        peers=5
INFO[01-31|10:06:17.984] [Downloader] Downloaded PoS Headers      now=16523999 blk/sec=0.000
INFO[01-31|10:06:47.983] [txpool] stat                            block=0 pending=0 baseFee=0 queued=0 alloc=102.3MB sys=380.9MB
INFO[01-31|10:06:47.984] [Downloader] Downloaded PoS Headers      now=16523999 blk/sec=0.000
INFO[01-31|10:06:47.984] [LightClient] P2P                        peers=6
INFO[01-31|10:07:17.984] [Downloader] Downloaded PoS Headers      now=16523999 blk/sec=0.000
INFO[01-31|10:07:46.122] [p2p] GoodPeers                          eth66=0 eth67=0
INFO[01-31|10:07:47.983] [txpool] stat                            block=0 pending=0 baseFee=0 queued=0 alloc=78.0MB sys=380.9MB
INFO[01-31|10:07:47.984] [LightClient] P2P                        peers=14
INFO[01-31|10:07:47.984] [Downloader] Downloaded PoS Headers      now=16523999 blk/sec=0.000
INFO[01-31|10:08:17.984] [Downloader] Downloaded PoS Headers      now=16523999 blk/sec=0.000
INFO[01-31|10:08:47.983] [txpool] stat                            block=0 pending=0 baseFee=0 queued=0 alloc=89.3MB sys=380.9MB
INFO[01-31|10:08:47.984] [Downloader] Downloaded PoS Headers      now=16523999 blk/sec=0.000
INFO[01-31|10:08:47.984] [LightClient] P2P                        peers=22
INFO[01-31|10:09:17.992] [Downloader] Downloaded PoS Headers      now=16523999 blk/sec=0.000
```

**Comment by dhyaniarun1993** [comment]
I was able to sync the client. Closing this.
